jupyter.py

```diff
@@ -0,0 +1,8 @@
+"""Launch the root jupyter command
+
+Alias to jupyter_core
+"""
+
```

Comment:
It would be clearer at present to release it as 4.0, but in the long run I think starting at 1.0 is probably clearer.